cmd/agent/app/run.go

```diff
@@ -278,6 +279,14 @@ func StartAgent() error {
 	}
 	log.Debugf("statsd started")
 
+	// Start SNMP-related services
+	if traps.IsEnabled() {
```

Comment:
The trap feature relies on logs, to avoid loopholes like traps enabled but logs disabled, we should check whether logs are enabled in `traps.IsEnabled()` and log a warning if we have logs disabled.

Reply:
Good point! I went for skipping starting the trap server entirely if logs are disabled. (Otherwise it'd be useless work for the Agent and I'm not sure we wouldn't run into issues w/ the server output channel filling up over time etc…)